clouddriver-kubernetes/src/main/java/com/netflix/spinnaker/clouddriver/kubernetes/provider/view/KubernetesJobProvider.java

```diff
@@ -46,12 +46,15 @@
   @Getter private final String platform = "kubernetes";
   private final AccountCredentialsProvider accountCredentialsProvider;
   private final KubernetesManifestProvider manifestProvider;
+  private final boolean detailedPodStatus;
 
   KubernetesJobProvider(
       AccountCredentialsProvider accountCredentialsProvider,
-      KubernetesManifestProvider manifestProvider) {
+      KubernetesManifestProvider manifestProvider,
+      @Value("${kubernetes.jobs.detailed-pod-status:false}") boolean detailedPodStatus) {
```

Comment:
Shouldn't it be true by default for backward compatibility?

Reply:
The new field `mostRecentPodName` in the response replaces the detailed pod statuses for what the UI needs, so it shouldn't have any difference in behavior unless there are pipelines grabbing statuses from the context with pipeline expressions (unlikely?).
Do you see this default may cause problems?

Reply:
I don't know for sure, but it's potentially used this way so I'd default to `true` with a note in the release notes that it may be switched to false in the future.